### PR TITLE
Peer Review submission

### PR DESCRIPTION
**Pull request by Herro-Sama**

### CODE REVIEW

File: Images/Ship'sBridge.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    def draw(self):
+        pos_x = self.pos_x
+        pos_y = self.pos_y
+        width = self.radius
+        height = self.height
+        Xwing = pygame.image.load('Xwing.png')
+
+
+
+def Gbolt(screen):
```
Comment (jack-maber) [inline Images/Ship'sBridge.py:84]: What does Gbolt mean?

File: Images/Ship'sBridge.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+            Red = screen.get_at((X, Y)).r
+            Green = screen.get_at((X, Y)).g
+            Blue = screen.get_at((X, Y)).b
+            pxarray[X, Y] = (255, Green/2, Blue/2)
+
+
+pygame.init()
+clock = pygame.time.Clock()
+
+window_width = 800
```
Comment (jack-maber) [inline Images/Ship'sBridge.py:112]: Constants should be in full caps I.E. HEIGHT

File: Images/Ship'sBridge.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+window_height = 600
+imgX = 0
+imgY = 0
```
Comment (jack-maber) [inline Images/Ship'sBridge.py:115]: x and y should be lowercase

File: Images/Wreckage.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+# Initialise PyGame
+pygame.init()
+clock = pygame.time.Clock()
+
+window_width = 1280
+window_height = 600
+window_size = (window_width, window_height)
+screen = pygame.display.set_mode(window_size)
+BackGround = Background('wreckage.jpg', [0,0])
```
Comment (jack-maber) [inline Images/Wreckage.py:103]: Random caps aren't good

File: Images/Biodome.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+sun = pygame.draw.circle(screen, white, (330, 140), 15, 0)
+
+pygame.init
+
+pygame.display.update()
+running = True
+while running:
+    for event in pygame.event.get():
+        if event.type ==pygame.QUIT:
+            running = False
```
Comment (jack-maber) [inline Images/Biodome.py:26]: Not very complex, only adds a circle to a reflection

File: Images/Ship'sBridge.py [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+
+        # Choose a random size
+        self.radius = random.randrange(1,5)
+        self.height = random.randrange(10,30)
+        # Choose a random colour
+        Brown1 = (244,164,96)
+        Brown2 = (210,180,140)
+        Brown3 = (222,184,135)
+        Brown4 = (255,228,196)
```
Comment (jack-maber) [inline Images/Ship'sBridge.py:36]: Brown's could be in a list rather than stacked up

File: Images/Biodome.py (hunk 5 above)
Comment (Sinjmaster) [reply Images/Biodome.py:26]: Agreed

File: Images/Biodome.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import pygame, sys, time
+
+
+#Colour
+white = (255,255,255)
+
+#Create window
+window_width = 1024
+window_height = 578
+screen = pygame.display.set_mode([window_width, window_height])
```
Comment (Stompyy) [inline Images/Biodome.py:10]: pep 8 capitals etc

File: Images/Ship'sBridge.py [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+        self.pos_y = random.randrange(250, 350)
+
+        speed = [-10,-5,5,10]
+
+        # Choose a random speed
+        self.speed_x = random.choice(speed)
+        self.speed_y = random.choice(speed)
+        if self.speed_x == 0:
+            self.speed_x = random.randrange(-10, 10)
+        if self.speed_x > 11:
```
Comment (Stompyy) [inline Images/Ship'sBridge.py:17]: if speed is only a random choice of -10, -5, 5, or 10, do any of these if statements apply?

File: Images/Ship'sBridge.py (hunk 6 above)
Comment (Stompyy) [reply Images/Ship'sBridge.py:36]: pep 8 capital letters for constants

File: Images/Ship'sBridge.py [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+        Brown4 = (255,228,196)
+        White = (255,255,255)
+        Colour = [White]
+        self.colour = (random.choice(Colour))
+
+    def update(self):
+
+        Tie = pygame.image.load('Tie.gif')
+        Xwing = pygame.image.load('Xwing.png')
+
```
Comment (Stompyy) [inline Images/Ship'sBridge.py:45]: pep 8 capitals

**Pull request closed without merging**